submissions/sinchUK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Popup</title>
+</head>
+<body>
+    <header class="menu">
+        <nav>
+            <ul class="menu-items">
+                <li class="menu-item">
+                    <a href="#">
+                        Mail
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">
+                        Images
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <div class="menu-item__inner">
+                        <input class="menu-item__input" type="checkbox"/>
+                        <img class="menu-item-img" src="./icons/popup-button.png" alt="submenu-icon"/>
+                        <div class="tooltip">
+                            <div class="tooltip-items">
+                                <div class="tooltip-item">
+                                    <a href="#">
+                                        <div class="tooltip-item-icon">
+                                            <img src="./icons/calendar.png" width="45" height="45" alt="Calendar"/>    
+                                        </div>
+                                        <span>Calendar</span>
+                                    </a>
+                                </div>
+                                <div class="tooltip-item">
+                                    <a href="#">
+                                        <div class="tooltip-item-icon">
+                                            <img src="./icons/mail.png" width="45" height="45" alt="Mail"/>
```

Comment:
[Read this article](https://jakearchibald.com/2022/img-aspect-ratio/) about handling image aspect ratio.
The highlighted element should be a circle.
![image](https://user-images.githubusercontent.com/16196199/190485689-0cac340f-0dc0-49af-a267-2ae956d52ac0.png)

Reply:
Resolved with "object-fit: contain".